### PR TITLE
Add Docker information

### PR DESCRIPTION
**Pull request by JLDLaughlin**
### Background
Currently, we describe how a user could add our `metabase-materialize-driver` plugin to a running instance of Metabase. 

### The Problem
Some people will want to run Metabase via Docker!

### The Solution
This PR adds:
- a simple Dockerfile that builds the latest version of Metabase, adds our plugin jar to its `/plugins` directory, and makes it executable by owners
- documentation on how to use the Dockerfile to build an image and spin up a container

### Future Work
Add more deployment details! Docker-compose, AWS, etc.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM metabase/metabase:latest
+ADD https://github.com/MaterializeInc/metabase-materialize-driver/releases/download/0.0.1/materialize-driver-0.0.1-SNAPSHOT-standalone.jar /plugins/
+RUN chmod 744 /plugins/materialize-driver-0.0.1-SNAPSHOT-standalone.jar
```
Comment (quodlibetor) [inline Dockerfile]: I think technically this should include a trailing newline. Stylistically I'd definitely prefer it if it did.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,60 @@
...
+| Port              | 6875      |
+| Database username | default   |
+| Database password | default   |
+
+
+### With Docker
+
+To generate a Metabase Docker image containing the Materialize driver,
+use [our Dockerfile](Dockerfile):
+```shell script
```
Comment (quodlibetor) [inline README.md]: this should probably use `Dockerfile` syntax, but the other option is just omitting this and leaving the link.

File: README.md [hunk 3]
```diff
@@ -3,7 +3,60 @@
...
+
+Once the Materialize driver is registered, use the following information
+to connect: 
+
+| Field             | Value     |
+| ----------------- |:---------:|
+| Host              | localhost |
+| Port              | 6875      |
+| Database username | default   |
+| Database password | default   |
```
Comment (quodlibetor) [inline README.md]: are these all the defaults? If so, should we maybe omit lines 15-23, and just tell people that the `materialize` database type should be available?

Comment (JLDLaughlin) [reply README.md]: these are the defaults! happy to remove

Comment (JLDLaughlin) [reply README.md]: I am nervous people won't trust the defaults.

**Comment by quodlibetor** [comment]
actually it might nice if we had a `bin/build.sh` that just has `build` and `publish` functions that take a tag and publishes something like `materialize/metabase-materialize-driver` to docker hub.

**Comment by quodlibetor** [comment]
or just two scripts, we don't need to add functions

**Comment by JLDLaughlin** [comment]
Okay, I will do that in a follow-up!

**Pull request merged**